marsappspa/src/ComponentPage.tsx

```diff
@@ -14,7 +14,7 @@ function ComponentPage() {
     );
   }
 
-function MyComponent(props: MyComponentProps) {
+export function MyComponent(props: MyComponentProps) {
     return (
       <div className='Component'>
         <h1>Title</h1>

```

Comment:
let's say you want different content for the components you have. take a look at https://felixgerschau.com/react-component-composition/